src/app/modules/group/components/member-list/member-list.component.html

```diff
@@ -54,6 +54,12 @@
           <ng-container *ngIf="col.field === 'user_count'">
             {{ rowData.userCount }}
           </ng-container>
+          <ng-container *ngIf="col.field === 'login'">
+            {{ rowData.login }}
+          </ng-container>
+          <ng-container *ngIf="col.field === 'parentGroups'">
```

Comment:
it becomes quite repetitive, you could do a switch (or if-elseif-else) on col.field, treat 'user.login', 'member_since' and 'user_count' separately, then on default do `{{ rowData[col.field] }}`

Reply:
fixed